{"uuid":"b18e3d63-3991-42fc-b677-0510cc28e1c8","historyId":"f47df1b837812c55122c7f448e6da5cf","fullName":"Session05__ValidateHTTP_ResponseHeader.ValidateResponseHeader_TESTCASE.GetSingleUser","labels":[{"name":"package","value":"Session05__ValidateHTTP_ResponseHeader.ValidateResponseHeader_TESTCASE"},{"name":"testClass","value":"Session05__ValidateHTTP_ResponseHeader.ValidateResponseHeader_TESTCASE"},{"name":"testMethod","value":"GetSingleUser"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"RestAssuredAPI"},{"name":"subSuite","value":"Session05__ValidateHTTP_ResponseHeader.ValidateResponseHeader_TESTCASE"},{"name":"host","value":"DESKTOP-LOFOO1Q"},{"name":"thread","value":"19336@DESKTOP-LOFOO1Q.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"GetSingleUser","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [application/json] but found [application/json; charset=utf-8]","trace":"java.lang.AssertionError: expected [application/json] but found [application/json; charset=utf-8]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:656)\r\n\tat org.testng.Assert.assertEquals(Assert.java:666)\r\n\tat Session05__ValidateHTTP_ResponseHeader.ValidateResponseHeader_TESTCASE.GetSingleUser(ValidateResponseHeader_TESTCASE.java:22)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\r\n\tat org.testng.TestRunner.run(TestRunner.java:621)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:336)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1200)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1114)\r\n\tat org.testng.TestNG.run(TestNG.java:1082)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1698839083722,"stop":1698839087117}